src/mocks.ts

```diff
@@ -6,81 +6,56 @@ const log				= logger(path.basename( __filename ), {
 });
 
 
-function ZomeAPIResult ( result ) {
-    return {
-	"Ok": result,
-    };
+function HhaResult ( result ) {
+    return { result };
```

Comment:
The result should not be in an object either. you can just `return result`
So we don't need this function anymore.